pkg/provisioner/block/block.go

```diff
@@ -125,7 +150,7 @@ func (block *blockProvisioner) Provision(options controller.VolumeOptions,
 			PersistentVolumeReclaimPolicy: options.PersistentVolumeReclaimPolicy,
 			AccessModes:                   options.PVC.Spec.AccessModes,
 			Capacity: v1.ResourceList{
-				v1.ResourceName(v1.ResourceStorage): options.PVC.Spec.Resources.Requests[v1.ResourceName(v1.ResourceStorage)],
+				v1.ResourceName(v1.ResourceStorage): capacity,
```

Comment:
I don't believe this works, as I mentioned in https://github.com/oracle/oci-volume-provisioner/issues/120#issuecomment-391545699 the discrepancy between the PV and PVC will result in the reconciler constantly attempting to re-provision the PV with the correct capacity.